doc/dependencies.md

```diff
@@ -16,3 +16,4 @@ These are the dependencies needed to build Zen on Linux.
 | [Libtool](https://www.gnu.org/software/libtool/) | [2.4.6] |
 | [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/) | [0.29.2] |
 | [hexdump](https://www.kernel.org/pub/linux/utils/util-linux/) | [2.37.2] |
+| [cmake](https://cmake.org/download/) | [3.22.1] |
```

Comment:
If this does not cause any issue or breaks any dependency build, I would lower the version requirement for CMake, as `apt` package manager on Ubuntu Focal by default installs version 3.16.3

Reply:
Done